.circleci/config.yml

```diff
@@ -19,10 +19,10 @@ ruby-docker-template: &ruby-docker-template
         fi
     - run: sudo apt-get update -y && sudo apt-get install -y build-essential
     - run: ruby -v
-    - run: gem install bundler -v "~> 1.17"
-    - run: bundle install
+    - run: gem install bundler -v 2.2.10
+    - run: bundle _2.2.10_ install
```

Comment:
Because in Ruby you can have more than one version of a gem installed, and the Docker image probably already has a different version of Bundler. So if you're invoking it as a command-line tool, it needs to know which version you mean. The underscore parameter is a convention of Bundler. https://stackoverflow.com/questions/56880223/using-multiple-bundler-versions